values/fetch_historical_values.py

```diff
@@ -0,0 +1,22 @@
+import csv
+
+class HistoricalValueFetcher:
+    def __init__(self):
+        list_of_values = []
+        path = "Github/Crazy_tr8/historical_data/XBTUSD_Q3.csv"
+        with open(path, 'r') as file_csv:
+            csv_reader = csv.reader(file_csv)
+            for self.row in csv_reader:
+                list_of_values.append(self.row)
```

Comment:
If we know that we need only first and second column of a row why we are storing entire row? Shouldn't we just insert two elements to our list_of_values?

```
   for row in csv_reader:
     self.list_of_values.append( (row[0], row[1])   )
```